### ISSUE TITLE
does genpasswd require --conf?

### ISSUE THREAD

**Issue by baranelitez** [ergochat/ergo#2191]
Hi. `./ergo genpasswd` requires `--conf ircd.yaml` to work. any reason behind this? or can I create a password on my local machine to copy the hashed password to another config running on another machine?

**Comment by slingamn** [comment]
I cannot reproduce this against 2.14.0:

```
shivaram@good-fortune:~$ ergo --version
ergo-v2.14.0
shivaram@good-fortune:~$ ls ircd.yaml
ls: cannot access 'ircd.yaml': No such file or directory
shivaram@good-fortune:~$ ergo genpasswd
Enter Password: 
Reenter Password: 
$2a$04$gRapvpnSFDmyhM/g238MueVG5cwH0sJJkBfYbctXHg0HgSCEnFz7S
```

**Comment by slingamn** [comment]
Regardless of this, hashed passwords generated for operators or WEBIRC should not depend on the config; you should be able to generate them on any machine and transfer them to your server config.

**Comment by slingamn** [comment]
AFAICT this was fixed in 2c7c8fbaf9ba431e97fd52b0a0fa260e60c7d5ed

**Comment by baranelitez** [comment]
it was fixed and it was due to a typo i made during testing. sorry
